src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/syntaxes/aspnetcorerazor.tmLanguage.json

```diff
@@ -237,9 +288,13 @@
       "end": "(\\})",
       "endCaptures": {
         "1": {
-          "name": "punctuation.curlybrace.close.cs"
+          "name": "keyword.control.razor.directive.codeblock.close"
         }
       }
+    },
```

Comment:
Just curious why this got moved, seems it didn't change otherwise?

Reply:
I think it's just a git diff mix up. Bad git bad!